locale/ne.js

```diff
@@ -58,39 +58,39 @@
                 return symbolMap[match];
             });
         },
-        meridiemParse: /राती|बिहान|दिउँसो|बेलुका|साँझ|राती/,
+        // We only have four state of time.
+        // साँझ and बेलुका are synonyms only!
+        meridiemParse: /राति|बिहान|दिउँसो|साँझ/,
```

Comment:
I could agree to this, if not being very pedantic. The comments could have been placed in the discussion itself. Maybe remove them from here.